.circleci/config.yml

```diff
@@ -109,9 +109,12 @@ commands:
         default: false
     steps:
       - build
+      - run:
+          name: Set npm default directory
+          command: bash scripts/set-npm-default-directory.sh
```

Comment:
I personally don't think we need to put this in a separate bash script. Also, we can use the whole CircleCI `BASH_ENV` thing instead of a `~/.profile` file.

Reply:
The build process also installs npm dependencies, then we change the default directory? Should we change the default directory before everything?